data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/rewards/GetAttestationRewards.java

```diff
@@ -0,0 +1,100 @@
+
+  private static final SerializableTypeDefinition<IdealAttestationReward> IDEAL_REWARDS_TYPE =
+      SerializableTypeDefinition.object(IdealAttestationReward.class)
+          .withField("effective_balance", LONG_TYPE, IdealAttestationReward::getEffectiveBalance)
+          .withField("head", LONG_TYPE, IdealAttestationReward::getHead)
+          .withField("target", LONG_TYPE, IdealAttestationReward::getTarget)
+          .withField("source", LONG_TYPE, IdealAttestationReward::getSource)
+          .build();
+
+  private static final SerializableTypeDefinition<TotalAttestationReward> TOTAL_REWARDS_TYPE =
+      SerializableTypeDefinition.object(TotalAttestationReward.class)
+          .withField("validator_index", LONG_TYPE, TotalAttestationReward::getValidatorIndex)
+          .withField("head", LONG_TYPE, TotalAttestationReward::getHead)
+          .withField("target", LONG_TYPE, TotalAttestationReward::getTarget)
+          .withField("source", LONG_TYPE, TotalAttestationReward::getSource)
+          .withField("inclusion_delay", LONG_TYPE, TotalAttestationReward::getInclusionDelay)
+          .build();
+
+  private static final SerializableTypeDefinition<AttestationRewardsData> DATA_TYPE =
+      SerializableTypeDefinition.object(AttestationRewardsData.class)
+          .withField(
+              "ideal_rewards",
+              SerializableTypeDefinition.listOf(IDEAL_REWARDS_TYPE),
+              AttestationRewardsData::getIdealAttestationRewards)
+          .withField(
+              "total_rewards",
+              SerializableTypeDefinition.listOf(TOTAL_REWARDS_TYPE),
+              AttestationRewardsData::getTotalAttestationRewards)
+          .build();
+
+  public static final SerializableTypeDefinition<GetAttestationRewardsResponse> RESPONSE_TYPE =
+      SerializableTypeDefinition.object(GetAttestationRewardsResponse.class)
+          .name("GetAttestationRewards")
+          .withField(
+              EXECUTION_OPTIMISTIC,
+              BOOLEAN_TYPE,
+              GetAttestationRewardsResponse::isExecutionOptimistic)
+          .withField(FINALIZED, BOOLEAN_TYPE, GetAttestationRewardsResponse::isFinalized)
+          .withField("data", DATA_TYPE, GetAttestationRewardsResponse::getAttestationRewardsData)
+          .build();
+
+  public GetAttestationRewards() {
+    super(
+        EndpointMetadata.post(ROUTE)
+            .operationId("getAttestationsRewards")
+            .summary("Get Attestations Rewards")
+            .description(
+                "Retrieve attestation reward info for validators specified by array of public keys or validator index. If no array is provided, return reward info for every validator.")
+            .tags(TAG_BEACON, TAG_REWARDS)
+            .pathParam(EPOCH_PARAMETER)
+            .requestBodyType(DeserializableTypeDefinition.listOf(STRING_TYPE))
+            .response(SC_OK, "Request successful", RESPONSE_TYPE)
```

Comment:
adding a metadata test for 200 will make it a lot simpler when you come to implementing too :)